### DISCUSSION TITLE
Are there alternatives to make perfect cuts to H265 files?

### DISCUSSION THREAD

**Discussion by ySergi** [mifi/lossless-cut#1951] [General]
This software is great... when it comes to making perfect cuts in h264/mp4 or VP9/webm files... but unfortunately, all the footage I deal with comes from BDRips, encoded in AV1/mkv or H265/mkv, and that's something LosslessCut is currently unable to process properly, which makes trying to make perfect cuts a nightmare, even with the "Smart cut" mode enabled, it doesn't work properly. So, I'm looking for alternatives that do work with this type of files, thanks.

**Comment by mifi** [comment]
you mean alternative apps? i don't know, but if you don't mind re-encoding, then I think most video editors can do perfect cuts. losslesscut might also add support for re-encoding in the future.

**Reply by ySergi** [reply]
Well, I actually care about not re-encoding, I forgot to clarify that.
Iirc, there was a tool and a MPV script that allows doing perfect cuts without re-encoding. But idk how it's called.
